include/aikido/common/ExecutorMultiplexer.hpp

```diff
@@ -47,7 +47,7 @@ class ExecutorMultiplexer final
 private:
   /// Mutex for the list of callbacks. The array of callbacks will be locked
   /// during it's modified and the callbacks are called.
-  std::mutex mMutex;
+  mutable std::mutex mMutex;
```

Comment:
This is part of a proposal for style convention, to have class member mutex to always be explicitly `mutable` irrespective of whether the class has `const` member functions. 

Open to removing the keyword if the style convention is not favorable.